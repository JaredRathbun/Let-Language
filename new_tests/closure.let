fun mult := (lambda x => (lambda y => x * y))

apply (apply mult 4) 5