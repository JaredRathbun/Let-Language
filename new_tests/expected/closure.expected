Parse successful!
Result: 20